drv/stm32h7-hash/src/lib.rs

```diff
@@ -0,0 +1,364 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! STM32H7 Hash low-level driver crate.
+
+#![no_std]
+
+use drv_hash_api::{HashError, SHA256_SZ};
+
+// Other SKUs in the STM32 line having the HASH block:
+// stm32{f21[57],f4{05,07,27,29,69},f7{45,65,x6,x7,x9},h7{47cm[47],53{,v},b3},l4x6}
+#[cfg(feature = "h753")]
+use stm32h7::stm32h753 as device;
+
+use userlib::*;
+use zerocopy::AsBytes;
+
+/// TODO: For the purpose of performance analysis, the hash driver could be
```

Comment:
The location of this comment makes it look like it's commenting `use core::mem::size_of`. I would move it up top to the module comment `//!`.